### PR TITLE
new  extract_SCE function

### PR DESCRIPTION
**Pull request by kzintas**

### CODE REVIEW

File: R/TreeCluster-class.R [hunk 1]
```diff
@@ -131,7 +131,8 @@ ClusterHierarchy <- function(hierarchy, col_regex=NULL, columns =NULL) {
     "ClusterHierarchy",
     hierarchy
   )
-  
+  #print(hierarchy)
+  hierarchy
   # tree <- TreeIndex(hierarchy)
   # rownames(tree) <- rownames(hierarchy)
   #
```
Comment (jkanche) [inline R/TreeCluster-class.R:138]: @kzintas why do we need this change ?

File: R/TreeVizApp-class.R [hunk 2]
```diff
@@ -45,112 +55,162 @@ TreeVizApp$methods(
     .self$server$wait_to_clear_requests()
     .delay_requests(.self$server)
     
+  },
+  
+  
+  #' @import umap
+  #' @import Rtsne
+  extract_SCE = function(datasource_name = "SCRNA_1") {
```
Comment (jkanche) [inline R/TreeVizApp-class.R:63]: not a requirement anymore

**Comment by jkanche** [comment]
@kzintas do switch to the `devel` branch of the repo. That should be the default going forward.

File: R/TreeCluster-class.R (hunk 1 above)
Comment (kzintas) [reply R/TreeCluster-class.R:138]: I think I just wanted to make sure I am returning the correct object

Comment (kzintas) [reply R/TreeCluster-class.R:138]: We don't need that

File: R/TreeVizApp-class.R (hunk 2 above)
Comment (kzintas) [reply R/TreeVizApp-class.R:63]: you mean the datasource_name parameter?

**Pull request closed without merging**